administrator/language/en-GB/com_users.ini

```
@@ -102,7 +102,7 @@ COM_USERS_ERROR_LEVELS_NOLEVELS_SELECTED="No View Permission Level(s) selected."
 COM_USERS_ERROR_NO_ADDITIONS="The selected user(s) are already assigned to the selected group."
 COM_USERS_ERROR_NOT_IN_GROUP="The selected user(s) are not in the selected group."
 COM_USERS_ERROR_ONLY_ONE_GROUP="A user must belong to at least one group."
-COM_USERS_ERROR_VIEW_LEVEL_IN_USE="You can't delete the view access level '%d:%s' because it is being used by content."
+COM_USERS_ERROR_VIEW_LEVEL_IN_USE="You can't delete the view access level '%d:%s' because it is being used in tables: %s."
```

```suggestion
COM_USERS_ERROR_VIEW_LEVEL_IN_USE="You can't delete the view access level '%d:%s' because it is being used in the database tables: %s."
```

Makes it a little more clearer what it refers to - same terminology as com_checkin
A complete beginner might think its an html table

---

thx